buildSettings.js.tmpl

```diff
@@ -0,0 +1,3 @@
+var buildSettings = {
```

Comment:
It's a little odd to me that manifest.json.tmpl is in the root, and I'd probably want to just put it in addon/, but for now I copied that approach and we can discuss it later. (If it went in addon/, then we'd want to use `web-ext build --ignore-files '*.tmpl'`)

Reply:
Sorry, I went w/ putting it in the root because I didn't want it to get bundled into the add-on when running `web-ext build`. I totally forgot they added the `--ignore-files` flag.

Reply:
Filed as #60